.github/actions/bump-skip/action.yml

```
@@ -0,0 +1,27 @@
+# This action must be done after the checkout action
+name: 'bump-skip'
+description: 'Set skip-out when we are doing a version bump'
```

Is this an old name copied from somewhere? I don't see any other reference to `skip-out` in this PR.

```suggestion
description: 'Set is-bump when we are doing a version bump'
```

---

Yes, likely a copy paste thing, should be updated to `is-bump`.